saga-demo/saga-k8s-resources/base/alpha.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
```

Comment:
Please add ASF License header here.

Reply:
I'll add the License header, btw, I'm considering reorganizing the folder structure into a base folder and demos folder. The base folder provide alpha-server and postgresql that all demos need. The demos folder provide the demo's services.

Now I've done the reorganization and provide the spring-demo, pls take a look at the latest commit.